babel.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   // Convert ESM -> CJS; Specifically for @polkadot >= v4 where ESM becomes default
   plugins: ['@babel/plugin-transform-modules-commonjs'],
+  compact: false,
```

Comment:
If we don't need this, then we should remove it imo just to not risk anything changing imo.

If we do need it OK to leave